utils/_context/_scenarios/__init__.py

```diff
@@ -441,6 +441,7 @@ def all_endtoend_scenarios(test_object):
             "DD_TRACE_HTTP_CLIENT_ERROR_STATUSES": "200-201,202",
             "DD_SERVICE": "service_test",
             "DD_TRACE_KAFKA_ENABLED": "false",  # Using Kafka as is the most common endpoint and integration(missing for PHP).
+            "DD_TRACE_KAFKAJS_ENABLED": "false",  # In Node the integration is kafkajs.
```

Comment:
Why is it different in node ? this name isn't specified in a RFC ?

Reply:
I don't know if there was a specific naming requirement in an RFC when this integration was implemented but node just refers to the integration internally as the npm package name:

https://www.npmjs.com/package/kafkajs